### DISCUSSION TITLE
Citing MicroPython

### DISCUSSION THREAD

**Discussion by tejalbarnwal** [micropython/micropython#11860] [General]
Is there a conventional format to cite Micropython in Bibtex format?

**Comment by Josverl** [comment]
If bibtex can handle python refs, then I would expect it to be able to handle micropython refs as well.

Could you give a description of what you mean to do?

**Reply by tejalbarnwal** [reply]
yeah, i want to cite the repo in a research paper

**Comment by rkompass** [comment]
You could reference the GitHub Repository. How that is done a comparison with references to other repositories might tell.
If you want to put the focus on the history: The [kickstarter site](https://www.kickstarter.com/projects/214379695/micro-python-python-for-microcontrollers) might be appropriate.

**Reply by tejalbarnwal** [reply]
okay, i will reference the github repo directly
